src/api/spec/features/webui/requests_spec.rb

```diff
@@ -163,7 +163,16 @@
 
   context 'review' do
     describe 'for user' do
-      skip
+      let(:reviewer) { create(:confirmed_user) }
```

Comment:
To avoid colliding names of projects in the backend, creating a user also creates the project...